### DISCUSSION TITLE
How can participatory mapping and critical cartography be improved?

### DISCUSSION THREAD

**Discussion by rachan2023** [albertkun/21S-ASIAAM-191A#58] [Past Thinking Caps]
Participatory mapping and critical cartography can be improved by acknowledging that the world political map is oppressive to marginalized communities, putting open-source GIS tools in the hands of the people, and encouraging people to use maps and map-making to challenge the societal norms set in place. Oftentimes, the world political map is held as the universal standard for maps, which is used for political leaders and institutions to assert their dominance and rewrite everyone’s narratives. As a result, people need to think critically about how and why the maps that we see today were created and its implications on marginalized communities. Alongside understanding the ethics of map-making, GIS and open-source map-making tools need to be put in the hands of the people in order to improve participatory mapping and critical cartology. When map-making tools are no longer exclusive to map-makers and those with power, people are able to use open-source map-making tools to collaborate with one another, create maps that reveal a different perspective and challenge societal norms, and craft their own narrative. By giving people the necessary tools and resources to create maps of resistance, the power is handed back to the people. 

An example is Aaron Carapella’s map of all the Native American tribes in North America, which I found in an NPR article. Carapella is a self-taught map-maker who created the map below because he could not find a map that showed all of the Native American tribes (since maps often erase the history of Native Americans). As a result, he created this map which directly challenges the U.S. political map and shows its oppressive nature as well as reveals the roots of Native American tribes. 

Link to article: npr.org/sections/codeswitch/2014/06/24/323665644/the-map-of-native-american-tribes-youve-never-seen-before

[Tribal_Nations_Map_NA.pdf](https://github.com/albertkun/21S-ASIAAM-191A/files/6290917/Tribal_Nations_Map_NA.pdf)

**Comment by ghost** [comment]
I think the points you made above are extremely important to take into consideration, especially concerning the fact that participatory mapping would help challenge misconceptions and narratives put forth by politically driven maps as you mentioned. Looking at the map you provided made me think back to Mariah Tso's lecture this week about how tribal names have been westernized and how often I see street/city names that misrepresent the Native American community. I really liked your phrase "maps of resistance" because I think that represents the very essence of participatory and critical cartography.

**Comment by nnhimichelle** [comment]
I really enjoyed reading your response, I've honestly never thought of it that way! I feel like with addressing the problems of using the world political map as the universal standard, it could then allow people to raise awareness and compel them to learn more about critical cartography and see the values in participatory mapping. With the inclusivity of resources for others, it would also create more engagement for more communities! And thank you for the article, it was a great read!
